### ISSUE TITLE
Touching a browsing context in an "in parallel" section is not OK.

### ISSUE THREAD

**Issue by jan-ivar** [w3c/mediacapture-main#488]
Some feedback from our DOM team on the [getUserMedia](https://w3c.github.io/mediacapture-main/getusermedia.html#dom-mediadevices-getusermedia()) algorithm's handling of `originIdentifier`:

1)  A browsing context does not have a concept of "origin".
2)  Touching a browsing context in an "in parallel" section is not OK.
3)  The top-level browsing context may not have anything to do with the API being invoked.

Basically, this algorithm needs to capture the origin from a sane place before going parallel.  Right now it's not doing any of those things.
